collections/_sdk/canvas_manifest.md

```diff
@@ -0,0 +1,737 @@
+{
+  "class": "my_plugin.handlers.patient_sync:PatientSync",
+  "description": "Syncs patient data with external system",
+  "data_access": {
+    "event": "patient.updated",
+    "read": ["Patient", "Coverage"],
+    "write": ["Patient"]
+  }
+}
+```
+
+### Content
+
+Content components inject custom content into specific areas of the Canvas UI.
+
+**Required Fields:**
+- `class` (string): Python path to your content provider class
+- `description` (string): What content this provides
+
+```json
+{
+  "class": "my_plugin.content.banner:CustomBanner",
+  "description": "Displays custom banner in patient chart"
+}
+```
+
+### Effects
+
+Effects are side-effect handlers that perform actions in response to events without returning data.
+
+**Required Fields:**
+- `class` (string): Python path to your effect handler class
+- `description` (string): What effect this produces
+
+```json
+{
+  "class": "my_plugin.effects.notification:SendNotification",
+  "description": "Sends notification when conditions are met"
+}
+```
+
+### Views
+
+Views provide custom data views or API endpoints that return data without side effects.
+
+**Required Fields:**
+- `class` (string): Python path to your view class
+- `description` (string): What data this view provides
+
+```json
+{
+  "class": "my_plugin.views.patient_summary:PatientSummaryView",
+  "description": "Returns summarized patient data"
+}
+```
+
+### Questionnaires
+
+Questionnaires define forms or surveys that can be presented to users.
+
+**Required Fields:**
+- `template` (string): Path to the questionnaire template file
+
+```json
+{
+  "template": "templates/intake_form.json"
+}
+```
+
+## Application Scope and Menu Positioning
+
+Understanding scope and menu positioning is crucial for creating applications that appear in the right place with the right behavior.
+
+### Scope Types
+
+The `scope` field determines where your application appears and what context it has access to.
+
+#### `patient_specific`
+
+**Description:** Application appears in the patient chart and has access to the currently selected patient's context.
+
+**Use Cases:**
+- Custom chart views showing patient-specific data
+- Clinical decision support tools that analyze patient information
+- Patient-specific calculators or risk assessments
+- Custom documentation interfaces tied to a specific patient
+
+**UI Location:** Typically appears in the patient chart navigation
+
+**Example Scenario:** You're building a risk calculator that needs access to a patient's vitals, medications, and lab results to calculate cardiovascular risk. This should use `patient_specific` scope.
+
+```json
+{
+  "class": "my_plugin.apps.risk_calculator:RiskCalculator",
+  "name": "CVD Risk Calculator",
+  "description": "Calculate cardiovascular disease risk",
+  "scope": "patient_specific",
+  "icon": "assets/heart.png"
+}
+```
+
+#### `global`
+
+**Description:** Application is available organization-wide, not tied to any specific patient context.
```

Comment:
Do we expect an application in the global context to ALSO show in patient specific contexts? I would assume yes, but am hearing some reports of this not working